src/selectors/__tests__/getChildResolvedPath.ts

```diff
@@ -0,0 +1,30 @@
+import getChildResolvedPath from '../getChildResolvedPath'
+import { initialState, rankThoughtsSequential, reducerFlow, pathToContext } from '../../util'
+import { Path, SimplePath } from '../../types'
+import { toggleContextView, setCursor } from '../../reducers'
+
+it('get child thoughtsResolved', async () => {
+
+  const parentThoughtsResolved = rankThoughtsSequential(['m', 'n', 'o', 'p']) as Path
+  const childSimplePath = rankThoughtsSequential(['a', 'b', 'p', 'o', 'q']) as SimplePath
```

Comment:
`/m` and `/a` can't have the same rank since they are both in the root context. Not relevant for this test, but invalid states could lead to inconsistencies later on.